shared/src/common/constants/file/file.ts

```diff
@@ -0,0 +1 @@
+export const UPLOAD_FILE_KEY = 'file';
```

Comment:
```suggestion
 const UPLOAD_FILE_KEY = 'file';
 
 export { UPLOAD_FILE_KEY };
```
export only at the bottom